grafana-init/README.md

```diff
@@ -42,6 +42,7 @@ Configuration
 | `GRAFANA_URL`      | `http://grafana:3000`  | Location of Grafana server      |
 | `GRAFANA_USERNAME` | `mini-mon`             | Agent Keystone username         |
 | `GRAFANA_PASSWORD` | `password`             | Agent Keystone password         |
+| `GRAFANA_USERS`    | `[{'user': GRAFANA_USERNAME, 'password': GRAFANA_PASSWORD, 'email': ''}]` | Agent Keystone users. Default datasource and dashboard are created for multiple users if this variable is set as, e.g. `"[{'user': 'mini-mon', 'password': 'password', 'email': ''}, {'user': 'username', 'password': 'password', 'email': ''}]"`.  **NOTE: Set this variable in string type** |
```

Comment:
Mention that it will overwrite setting up  `GRAFANA_USERNAME` and `GRAFANA_PASSWORD`.

Reply:
Sorry for late update.
I added the description that default value is overwritten by other variables.